x/icacallbacks/keeper/keeper.go

```diff
@@ -87,37 +88,58 @@ func (k *Keeper) ClaimCapability(ctx sdk.Context, cap *capabilitytypes.Capabilit
 	return k.scopedKeeper.ClaimCapability(ctx, cap, name)
 }
 
-func (k Keeper) CallRegisteredICACallback(ctx sdk.Context, modulePacket channeltypes.Packet, txMsgData *sdk.TxMsgData) error {
+func (k Keeper) GetCallbackDataFromPacket(ctx sdk.Context, modulePacket channeltypes.Packet, callbackDataKey string) (cbd *types.CallbackData, found bool) {
```

Comment:
per stackoverflow https://stackoverflow.com/questions/15089726/why-would-return-parameters-be-named and a tour of go https://go.dev/tour/basics/7, one of the benefits is documenting the meaning of the return values 

going to leave this in for clarity

Reply:
ah nice! Agreed for `found`, but I'd argue `cdb` doesn't provide any clarity though since it's never actually used in the function, and provides no additional info over `types.CallbackData`